src/parsers/steps.ts

```diff
@@ -50,9 +50,9 @@ export function parseSteps(steps, definitions) {
             }
 
             const stepArgs = [
-                ...Array.from(
-                    definition.expression?.regexp?.exec(step.text) || []
-                ).slice(1),
+                ...definition.expression
```

Comment:
I'll add a nullish coalesce here since spread operator on an array will throw an error when the spread value is undefined / null.

https://github.com/tc39/ecma262/issues/687